metadata-ingestion/src/datahub/cli/docker.py

```diff
@@ -39,7 +39,9 @@
 
 BOOTSTRAP_MCES_FILE = "metadata-ingestion/examples/mce_files/bootstrap_mce.json"
 
-GITHUB_BASE_URL = "https://raw.githubusercontent.com/datahub-project/datahub/master"
+#GITHUB_BASE_URL = "https://raw.githubusercontent.com/datahub-project/datahub/master"
```

Comment:
Did you maybe forgot to remove this?

Reply:
It's a work in progress. Thank you for the note though!